Ansible/roles/marvin/templates/test_data.py.j2

```diff
@@ -326,13 +326,21 @@ test_data = {
             "StaticNat": "VirtualRouter"
         }
     },
+    "nw_off_L2_persistent": {
+        "name": 'Test L2 Network Offering persistent',
+        "displaytext": 'Test L2 Network Offering persistent',
+        "guestiptype": 'L2',
+        "traffictype": 'GUEST',
+        "ispersistent": 'True',
+        "specifyVlan": 'True'
+    },
     "network_offering_vlan": {
     "name": "Test Network offering",
     "displaytext": "Test Network offering",
     "guestiptype": "Isolated",
     "supportedservices": "Dhcp,Dns,SourceNat,PortForwarding",
     "traffictype": "GUEST",
-    "specifyvlan": "False",
+    "specifyVlan": "False",
```

Comment:
@Pearl1594 could change in the keyname (all lowercase to camelCase) break anything?

Reply:
@rhtyd It is specifyVlan as per base.py code for NetworkOffering:
```
        if "specifyVlan" in services:
            cmd.specifyVlan = services["specifyVlan"]
```
Hence made the change to this pre-existing info - all other places have it named correctly